docs/src/openapi-v1.yaml

```diff
@@ -273,6 +273,57 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /blocks/{blockId}/extrinsics/{extrinsicIndex}:
+    get:
+      tags:
+      - blocks
+      summary: Get an extrinsic by its extrinsicIndex and block height or hash. 
+      The pair blockId, extrinsicIndex is sometimes referred to as a Timepoint.
+      description: Returns a single extrinsic.
+      operationId: getExtrinsicByExtrsinsicIndex
```

Comment:
```suggestion
      operationId: getExtrinsicByIndex
```

Reply:
Yea I like this as well. Will change throughout the code.